resources/assets/js/components/Application/Skills/Skills.tsx

```diff
@@ -64,7 +82,7 @@ const Sidebar: React.FC<SidebarProps> = ({ menuSkills, intl, status }) => {
       </p>
       <ul>
         {Object.keys(menuSkills).map((skillId) => (
-          <li key={skillId}>
+          <li key={`application-skills-sidebar-skill-${skillId}`}>
```

Comment:
I don't know that it's necessary to be so verbose with keys, see https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings

Reply:
Yeah, you're right. I think I was confused about what I was trying to fix when I did this :P I'll reverse it.